plutus-playground-lib/src/Playground/Types.hs

```diff
@@ -86,6 +77,37 @@ data SimulatorWallet =
     deriving (Show, Generic, Eq)
     deriving anyclass (ToJSON, FromJSON)
 
+data ContractCall a
+    = AddBlocks
+          { blocks :: Int
+          }
```

Comment:
IMO there should also be a constructor

```haskell
PayToWallet
    { sender :: Wallet
    , recipient :: Wallet
    , amount :: Value
    }
```

right now this is shoe-horned into `CallEndpoint`, but it doesn't belong there because it's not a function of the contract

Reply:
Thanks for the suggestion. Added!